### DISCUSSION TITLE
Unable to trace a method with a parameter by using MethodInstrumentationModule

### DISCUSSION THREAD

**Discussion by harveychong** [open-telemetry/opentelemetry-java-instrumentation#9609] [Q&A]
I am unable to trace a method with one parameter ( `com.my.processor.MyEvent` ) in my class (`com.my.processor.BlobProcessor`) implementing `java.util.function.Consumer` by using `MethodInstrumentationModule` but this module works well for my own method without any input parameter.  The line below is my configuration:

`otel.instrumentation.methods.include=com.my.processor.BlobProcessor[accept]`

Can someone help me about it?  

You help will be appreciated.

**Comment by laurit** [comment]
@harveychong please provide a sample application that allows reproducing this

**Comment by harveychong** [comment]
Thank you very much for your quick reply.  

I followed your instructions to create a sample application to reproduce this but I failed as the extension always works.  So back to our application, I finally found I created  a checkpoint in db before and so the monitored method was never called.   

I sorry to bother you today for my stupid mistake.
